### DISCUSSION TITLE
Scores were recalculated but shows the old score.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [ppy/osu#25914] [Q&A] [answered]
when lauching osu!lazer 2023.1220.0 my scores were recalculated

but now if I play maps I had scores on recently I get better scores than before even tho I miss more, and so I tried playing one of the replays with old score and the result screen showed completely different score than the once that is saved.

I assume the score was not recalculated succefully and it just shows the old score but diring replay it calculates the new one.

**Comment by bdach** [comment, marked as answer]
Only stable scores were supposed to be recalculated. Past lazer scores were not.

Does this answer the question?

**Reply by KoleckOLP** [reply]
Well that is a shame so my older lazer plays can't really be compared with new ones?
that kinda sucks I liked comparing my playes so I have to start doing that all over?

**Reply by KoleckOLP** [reply]
I had no stable scores in my lazer as far as I know but still some recalculations happened.

**Reply by bdach** [reply]
Yes the notification is a touch misleading but it is what it is.

And yes we will not offer recalculations to past local lazer scores at this stage as the support burden of maintaining the upgrade path is prohibitively expensive.

**Comment by BlauFx** [comment]
>And yes we will not offer recalculations to past local lazer scores at this stage as the support burden of maintaining the upgrade path is prohibitively expensive.

At which date/lazer version are scores considered not "past local lazer scores"?

**Reply by peppy** [reply]
you can check the changelogs to figure this out. but do note that we didn't have local score versioning until yesterday.
